asana-favicon-notification.js

```diff
@@ -52,7 +52,10 @@
 
   const waitForSideBar = () => (new Promise((resolve, reject) => {
     const loadedSidebar = document.querySelector('#asana_sidebar');
-    if (loadedSidebar) return Promise.resolve(loadedSidebar);
+    if (loadedSidebar) {
+      console.debug('=>> Found sidebar already present.', { loadedSidebar });
+      return resolve(loadedSidebar);
```

Comment:
this is the bug. Instead of calling `resolve`, I was returning a new resolved Promise. Probably should just convert this to `async/await`.